BeVolt/BSP/Simulator/DataGeneration/Strobelight.py

```diff
@@ -0,0 +1,19 @@
+import csv
+import os
+#This module holds status of strobe (on/off)
+
+#path name to file
+file = "BSP/Simulator/DataGeneration/Data/Strobe.csv"
+
+#returns integer where 1 means strobe is on, 0 means strobe is off
+def read();
+    strobe = 0
+    os.makedirs(os.path.dirname(file), exist_ok=True)
+    with open(file, 'a') as csvfile:
+        pass   #creates file if it doesn't exist
+    with open(file 'r') as csvfile: #read file
+        csvreader = csv.reader(csvfile)
+        if len(strobe):
+            return int (strobe)
+        else:
+            return 0
```

Comment:
You never actually update the value of strobe, so this won't tell us anything about the actual state of the strobelight.

Reply:
The point is to read the value that’s written in the CSV file. 

Right now, you set strobe to 0, do some stuff with a CSV file that ultimately doesn’t affect anything, then return int(strobe) which will always be 0 since you never actually read the value in csvreader no to the strobe variable.